go/vt/vtctl/workflow/server.go

```diff
@@ -1837,6 +1838,16 @@ func (s *Server) VDiffCreate(ctx context.Context, req *vtctldatapb.VDiffCreateRe
 		span.Annotate("auto_start", req.GetAutoStart())
 	}
 
+	var err error
+	req.Uuid = strings.TrimSpace(req.Uuid)
+	if req.Uuid == "" { // Generate a UUID
+		req.Uuid = uuid.New().String()
```

Comment:
This is the correct way to use uuid according to go docs, but I see that in vdiff2.go we are doing something different.
```
vdiffUUID, err = uuid.NewUUID()
```
We probably need to audit our usage of `New()` vs `NewUUID()` separately and make them all uniform.

Reply:
Nice catch! It's only in the deprecated legacy client that we use `NewUUID()` which is v1 whereas `New()`/`NewString()`/`New().String()` is v4. All of the currently supported code is using v4. And given that VDiff doesn't care — it only serves as a unique identifier — the two can coexist fine on the server side.